lib/generators/utils/entry.js

```diff
@@ -84,7 +84,11 @@ module.exports = (self, answer) => {
 					validate
 				)
 			])
-			.then(singularAnswer => `"${singularAnswer["singularEntry"]}"`);
+			.then(singularEntryAnswer => {
+				let { singularEntry } = singularEntryAnswer;
```

Comment:
Why is this needed?

Reply:
You mean destructuring or double quotes support?

Reply:
Looking at the test, probably to check if entry point name has `\` ?